charts/sonarqube-dce/Chart.yaml

```diff
@@ -24,6 +24,8 @@ maintainers:
     email: jeremy.cotineau@sonarsource.com
 annotations:
   artifacthub.io/changes: |
+    - kind: fixed
+      description: "Fix install_plugins.sh not deleting previously installed plugins"
     - kind: changed
```

Comment:
nope, we shouldn't remove them. Those are all the changes that we made since the last release (they do not include past things). If we maintain this list (adding a few lines here) every time we make a change, it will avoid extra work when doing the release.

We should actually update our contribution guidelines to make sure that people know about the new policy of adding an entry here in addition to the changelog...